scopes/workspace/workspace/workspace.ts

```diff
@@ -1277,6 +1280,73 @@ export class Workspace implements ComponentFactory {
       await this.importObjects();
       this.logger.consoleSuccess();
     }
+    return this.installModules(options);
+  }
+
+  private async getComponentsWithDependencyPolicies() {
+    const allComponentIds = await this.listIds();
+    const componentConfigFiles: Record<string, ComponentConfigFile> = {};
+    const componentPoliciesById: Record<string, any> = {};
+    (
+      await Promise.all<ComponentConfigFile | undefined>(
+        allComponentIds.map((componentId) => this.componentConfigFile(componentId))
+      )
+    ).forEach((component) => {
+      if (component == null) return;
+      const depResolverAspect = component?.aspects.get(DependencyResolverAspect.id);
+      if (depResolverAspect == null || component == null) return;
+      const componentId = component.componentId.toString();
+      componentConfigFiles[componentId] = component;
+      // eslint-disable-next-line
+      componentPoliciesById[componentId] = depResolverAspect.legacy.rawConfig['policy'];
```

Comment:
you should be able to get this using `depResolverAspect.config['policy']`  without accessing the legacy prop